src/views/AgmExtension.vue

```diff
@@ -206,7 +206,7 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
   @Getter(useRootStore) getUserInfo!: any
   @Getter(useBusinessStore) isGoodStanding!: boolean
   @Getter(useRootStore) isRoleStaff!: boolean
-
+  @Getter(useFilingHistoryListStore) getTotalAgmExtensionDuration!: number
```

Comment:
Super small thing James, we usually leave an empty line between the getters and local variables.

Reply:
Done